server/db/store.go

```diff
@@ -66,6 +74,14 @@ func NewStore(settings model.SqlSettings, wConnector, rConnector driver.Connecto
 		log:        log,
 	}
 
+	if *settings.DriverName == model.DatabaseDriverPostgres {
+		binaryParams, err := hasBinaryParams(*settings.DataSource)
+		if err != nil {
+			return nil, fmt.Errorf("failed to check binary parameters")
+		}
+		st.binaryParams = binaryParams
+	}
```

Comment:
I am testing because during development everything was working until I tried it on a real instance. See [the above comment](https://github.com/mattermost/mattermost-plugin-calls/pull/666#discussion_r1543780482) for context. Just want to make sure we test the store in both configurations.